go/adbc/driver/snowflake/statement.go

```diff
@@ -499,41 +523,56 @@ func (st *statement) ExecuteQuery(ctx context.Context) (array.RecordReader, int6
 		st.streamBind = nil
 
 		rdr := concatReader{}
-		err := rdr.Init(&bind)
+		err = rdr.Init(&bind)
```

Comment:
you named the record reader, lets be consistent and use `reader = concatReader{}` instead of `rdr := concatReader{}`